Temp variant until next LibreSSL release.
Taken-from: nextcloud/desktop/issues/738 @github

--- src/libsync/clientsideencryption.cpp.orig	2019-11-04 21:48:42 UTC
+++ src/libsync/clientsideencryption.cpp
@@ -36,6 +36,17 @@
 
 #include "wordlist.h"
 
+#ifdef LIBRESSL_VERSION_NUMBER
+#ifndef EVP_PKEY_CTX_set_rsa_oaep_md
+# define EVP_PKEY_CTRL_RSA_OAEP_MD	(EVP_PKEY_ALG_CTRL + 9)
+# define EVP_PKEY_CTRL_GET_RSA_OAEP_MD	(EVP_PKEY_ALG_CTRL + 11)
+
+# define EVP_PKEY_CTX_set_rsa_oaep_md(ctx, md) \
+	EVP_PKEY_CTX_ctrl(ctx, EVP_PKEY_RSA, EVP_PKEY_OP_TYPE_CRYPT,  \
+			  EVP_PKEY_CTRL_RSA_OAEP_MD, 0, (void *)(md))
+#endif
+#endif
+
 QDebug operator<<(QDebug out, const std::string& str)
 {
     out << QString::fromStdString(str);
